end-to-end-tests/src/test/java/io/eventuate/examples/tram/sagas/ordersandcustomers/endtoendtests/CustomersAndOrdersE2ETest.java

```diff
@@ -112,8 +118,25 @@ public void shouldSupportOrderHistory() {
     });
   }
 
+  @Test
+  public void testSwaggerUiUrls() throws IOException {
+    testSwaggerUiUrl(8081, "swagger-ui/index.html");
+    testSwaggerUiUrl(8082, "swagger-ui/index.html");
+    testSwaggerUiUrl(8083, "swagger-ui.html");
```

Comment:
Why is this different? Shouldn't the paths be consistent?

Reply:
swagger-ui/index.html - autogenerated for spring controllers
swagger-ui.html - is created manually here: 

https://github.com/eventuate-tram/eventuate-tram-sagas-examples-customers-and-orders/blob/4914d49636be3e54ae616003d65019f0ed389ac1/api-gateway/src/main/java/io/eventuate/examples/tram/sagas/ordersandcustomers/apigateway/common/SwaggerController.java#L8-L14

if change @GetMapping("/swagger-ui.html") to @GetMapping("swagger-ui/index.html") it will not work, because for some reason swagger css and js are served from server root (not from <server-url>/swagger-ui/*)

I did not investigate it much, because I did not think that is important.

I will investigate it then